.env

```diff
@@ -2,3 +2,4 @@ ARWEAVE_URL=https://arweave.net/
 IPFS_CDN=https://ipfs.cache.holaplex.com/
 ARWEAVE_CDN=https://arweave.net/
 ASSET_PROXY_ENDPOINT=https://assets.holaplex.com/
```

Comment:
We should put the delimiter into the asset_proxy.

`ASSET_PROXY_ENDPOINT=https://assets%d.holaplex.com/`

Reply:
It's worth noting dynamic format strings in Rust are not possible, so it would be better to use a descriptive and unique placeholder string for search-and-replace.

Reply:
It's worth noting dynamic format strings in Rust are not possible, so it would be better to use a descriptive and unique placeholder string for search-and-replace.

Reply:
we still need to add the ASSET_PROXY_COUNT or similar to scale without the need to touch the code

Reply:
Yeah, i think having `ASSET_PROXY_COUNT` and some kind of template string for `ASSET_PROXY_ENDPOINT` would be the ideal solution.

Reply:
place holder added. Also the marketplaces is broken due to the double forward slash in the urls. this commit fixes that as well